### PR TITLE
chore: switch algoexplorer urls to allo ENG-198

### PR DESCRIPTION
**Pull request by neilcampbell**
We will need to wait until Allo is launched before merging this PR.
It's still not clear that testnet version will be `http://testnet.allo.info`, will confirm once launched.

Changes are based on https://twitter.com/alloinfo/status/1737906303660347567?s=46&t=mW6Wsh-5XrgNlmfdOK46Fw

### CODE REVIEW

**Review by aorumbayev — APPROVED** [review]

**Review by robdmoore — APPROVED** [review]

**Comment by robdmoore** [comment]
@ehanoc @PhearZero @ori-shem-tov @HashMapsData2Value will let you merge and deploy from here.

**Comment by HashMapsData2Value** [comment]
https://twitter.com/Urthoo/status/1740366706021974445?t=mY-8QM5qUIxV6zIWEnnlsw&s=19

Seems like testnet 1) might not be prioritized, 2) if it's there the url would be allo.info/testnet/tx/...

**Comment by neilcampbell** [comment]
@HashMapsData2Value Thanks for the info!
Unfortunately for testnet we don't have many options as Pera explorer doesn't support transactions.

**Comment by robdmoore** [comment]
In the meantime we could use dappflow with a network switch and redirect - it's not very elegant, but it would work...

**Comment by neilcampbell** [comment]
@robdmoore Yeah I think that is the best option at this stage.
BTW url for that would be `https://app.dappflow.org/setnetwork?name=algonode_testnet&redirect=explorer/transaction/{txn_id}`. Pera explorer can be used to view account/address.

File: src/dapp/.env.template [hunk 1]
```diff
@@ -10,9 +10,9 @@ VITE_ALGOD_NETWORK=
 VITE_ENVIRONMENT=local
 VITE_IPFS_GATEWAY_URL=http://localhost:3000/ipfs
 
-VITE_ALGO_EXPLORER_URL=https://testnet.algoexplorer.io
+VITE_ALGO_EXPLORER_URL=https://testnet.allo.info
```
Comment (HashMapsData2Value) [inline src/dapp/.env.template:13]: Seems like it would be allo.info/testnet/... in the future

**Comment by neilcampbell** [comment]
@HashMapsData2Value I've updated the PR to use dappflow and pera on testnet.

As part of merging this (assuming you're happy with it), you'll also need to update the values for `ALGO_EXPLORER_URL` and `NFT_EXPLORER_URL` in the GitHub environment config for both dev and prod.

**Dev** (assuming it's pointed to testnet)
ALGO_EXPLORER_URL: https://app.dappflow.org/setnetwork?name=algonode_testnet&redirect=explorer
NFT_EXPLORER_URL: https://testnet.explorer.perawallet.app/assets/

**Prod**
ALGO_EXPLORER_URL: https://allo.info
NFT_EXPLORER_URL: https://allo.info/asset/

**Comment by ori-shem-tov** [comment]
Thanks @neilcampbell @robdmoore @HashMapsData2Value 
We also need to update env vars for `Dev` and `Prod`. I see that `NFT_EXPLORER_URL = https://nftexplorer.app/asset/` for `Prod`. I guess we want to keep it right?

**Comment by neilcampbell** [comment]
@ori-shem-tov Oh cool, yep you can keep that value as is then.

**Comment by HashMapsData2Value** [comment]
I replaced the testnet links with Pera Explorer's testnet instead of relying on Dappflow

**Comment by neilcampbell** [comment]
@HashMapsData2Value Awesome. It's nice that Pera supports viewing transactions and blocks now. I'll update the `algokit-cli` as well.

**Comment by ori-shem-tov** [comment]
@HashMapsData2Value I realized Pera explorer has no `/account` endpoint.
For account we need to use `/address` or `/accounts` (which redirects to `/address`). Let me see if we can ask them to change it, otherwise we'll have to support different endpoints for TestNet and MainNet

**Comment by neilcampbell** [comment]
@ori-shem-tov @HashMapsData2Value You could use Pera for both TestNet and MainNet and that way it'd be consistent. We ended up doing that in AlgoKit cli.

**Comment by robdmoore** [comment]
It would be great if the paths across the explorers were consistent, so if you can influence Pera to add support for account that would be awesome

**Comment by HashMapsData2Value** [comment]
https://explorer.perawallet.app/address/7RVD7JLRMHLWWSHSZBNQOZTVBCMZNEJIJC5JUVBEGWFT4MJZ3ETO5Z62QY/

https://allo.info/address/7RVD7JLRMHLWWSHSZBNQOZTVBCMZNEJIJC5JUVBEGWFT4MJZ3ETO5Z62QY

[allo.info](http://allo.info/) now redirects address -> account meaning we can use the same unified way

**Comment by HashMapsData2Value** [comment]
Returned the paths to address from account

          <Link href={`${import.meta.env.VITE_ALGO_EXPLORER_URL}/address/${address}`} target="_blank" className="font-normal">

**Review by HashMapsData2Value — APPROVED** [review]
Since Allo and Pera both now accept the /address/ this PR is basically just adding Pera explorer to testnet + some formatting fixes + dealing with some package.json/old packages vulnerabilities

**Pull request merged**